### DISCUSSION TITLE
Transfer AndydeCleyre/asdf-gat

### DISCUSSION THREAD

**Discussion by AndydeCleyre** [mise-plugins/.github#18] [Transfer Request]
Hello! I just made a [gat](https://github.com/koki-develop/gat/) plugin using the asdf template. My very minimal testing on Linux is successful, as is the macos testing done via github actions.

Could an owner take a look and see what changes if any might be required to get this adopted here? Thanks!

**Comment by jdx** [comment]
I'm not accepting any new asdf tools. Check to see if it can be installed via aqua/ubi.

**Reply by AndydeCleyre** [reply]
Ok, thanks. I was following the docs from https://mise.jdx.dev/plugins.html .

But now looking around for "ubi" and "aqua" I've found https://mise.jdx.dev/dev-tools/backends/ubi.html and will give it a try!

**Reply by AndydeCleyre** [reply]
So yes this works fine, thanks:

```console
$ mise use -g ubi:koki-develop/gat@latest
```
